catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java

```diff
@@ -115,8 +117,7 @@
     String output = Paths.get(System.getProperty("ddf.home"), FILE_NAMER.get())
             .toString();
 
-    @Option(name = "--delete", required = true, aliases = {
-            "-d"}, multiValued = false, description = "Delete Metacards and content after export. EG: --delete=true or --delete=false")
+    @Option(name = "--delete", required = true, multiValued = false, description = "Delete Metacards and content after export. EG: --delete=true or --delete=false")
```

Comment:
Instead of having to specify `--delete=true` or `--delete=false`,  is there a way to refactor this to simply be `--delete` in the case where we want to delete Metacards and content after export, otherwise don't delete when no flag is specified?